web/templates/login_to_view_image_button.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="spectator_login_for_image_button">
     <a class="login_button color_animated_button" href="/login/">
         <i class='zulip-icon zulip-icon-log-in'></i>
-        <span>{{t "Log in to view image" }}</span>
+        <span class="button-text">{{t "Log in to view image" }}</span>
```

Comment:
Maybe we can use a better class here like `login-text` but we use `color_animated_button` as parent CSS selector so just used a more general class here.

Reply:
This is the wrong philosophy. We want our class names to be unique across the app by itself, and not use a parent selector at all. [Previous discussion](https://chat.zulip.org/#narrow/stream/6-frontend/topic/CSS.20nesting/near/1283355).

Reply:
Yeah, `.color-animated-button-text` would be quite satisfactory for this.

Reply:
Will keep in this mind now.